examples/03_methanol/co2_hydrogenation_doc/tech_config_co2h.yaml

```diff
@@ -49,7 +49,7 @@ technologies:
     performance_model:
       model: "simple_generic_storage"
     cost_model:
-      model: "atb_battery_cost"
+      model: "generic_storage_cost"
```

Comment:
I like this change. values can come from other places than the atb, and for other techs